### ISSUE TITLE
Redux persistance in local storage

### ISSUE THREAD

**Issue by bbrock25** [spartansystems/rn-booster-kit#7]
There is a thing [redux-persist](https://github.com/rt2zz/redux-persist) that does this for us and supports react native. Its used in the [facebook f8 app](https://github.com/fbsamples/f8app/blob/master/js/store/configureStore.js) and seems pretty legit. 

We could actually use our apps in offline mode :)

**Comment by jasontwong** [comment]
Sounds like custom middleware would be important here to detect if there's connection or checking for staleness.

**Comment by bbrock25** [comment]
Yeah, I'm also conflicted cause there will be some loose schema to persist if we do this, not sure if this would require migrations or to just figure out a way to drop the dang thing and refresh.
